nvi-commons/src/main/resources/shape.ttl

```diff
@@ -0,0 +1,207 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix : <https://nva.sikt.no/ontology/publication#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:PublicationShape a sh:NodeShape ;
+  sh:targetClass :Publication ;
+  sh:property [
+    sh:path :contributor ;
+    sh:minCount 1 ;
+    sh:nodeKind sh:IRI
+  ] ;
+  sh:property [
+    sh:path :identifier ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:string
+  ] ;
+  sh:property [
+    sh:path :modifiedDate ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:dateTime
+  ] ;
+  sh:property [
+    sh:path :abstractText ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:string
+  ] ;
+  sh:property [
+    sh:path :isApplicable ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:boolean ;
+    sh:in ( "true"^^xsd:boolean )
+  ] ;
+  sh:property [
+    sh:path :isInternationalCollaboration ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:boolean
+  ] ;
+  sh:property [
+    sh:path :language ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:nodeKind sh:IRI
+  ] ;
+  sh:property [
+    sh:path :pageCount ;
+    sh:minCount 1 ;
+    sh:maxCount 1
+  ] ;
+  sh:property [
+    sh:path :publicationChannel ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:nodeKind sh:IRI
+  ] ;
+  sh:property [
+    sh:path :publicationDate ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:nodeKind sh:BlankNode
+  ] ;
+  sh:property [
+    sh:path :publicationType ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:nodeKind sh:IRI ;
+    sh:in ( :AcademicArticle )
+  ] ;
+  sh:property [
+    sh:path :status ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:nodeKind sh:IRI ;
+    sh:in ( :PUBLISHED )
+  ] ;
+  sh:property [
+    sh:path :title ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:string
+  ] ;
+  sh:property [
+    sh:path :topLevelOrganization ;
+    sh:minCount 1 ;
+    sh:nodeKind sh:IRI
+  ] .
+
+:PublicationDateShape a sh:NodeShape ;
+  sh:targetClass :PublicationDate ;
+  sh:property [
+    sh:path :year ;
+    sh:minCount 1 ;
+    sh:maxCount 1 ;
+    sh:datatype xsd:string ;
+    sh:pattern "^[0-9]{4}"
+  ] .
+
+
+:ContributorShape a sh:NodeShape ;
```

Comment:
This logs warnings about unaffiliated contributors missing an IRI, but this is expected and allowed. Can we change this to remove that warning? Example:


```
Node=<https://api.sandbox.nva.aws.unit.no/publication/0195c6f37392-68057afa-9b9f-4e7a-8c9a-f5aef6b657be>
  Path=<https://nva.sikt.no/ontology/publication#contributor>
  Value: _:Ba885a21b44ad40e6d5977fac50d13100
  Message: NodeKind[IRI] : Expected IRI for _:Ba885a21b44ad40e6d5977fac50d13100
  ```